examples/nidcpower_source_dc_voltage/_nidcpower_helpers.py

```diff
@@ -0,0 +1,104 @@
+"""nidcpower Helper classes and functions for MeasurementLink examples."""
+
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, TypeVar
+
+import click
+import grpc
+import nidcpower
+
+import ni_measurementlink_service as nims
+from ni_measurementlink_service import session_management
+from ni_measurementlink_service.measurement.service import MeasurementService
+
+# To use a physical NI SMU instrument, set this to False or specify
+# --no-use-simulation on the command line.
+USE_SIMULATION = True
+
+
+def create_nidcpower_session(
+    measurement_service: MeasurementService,
+    pin_names: Iterable[str],
+) -> Tuple[Any, List[session_management.SessionInformation]]:
+    """Create and reserve nidcpower sessions."""
+    session_management_client = nims.session_management.Client(
+        grpc_channel=measurement_service.get_channel(
+            provided_interface=nims.session_management.GRPC_SERVICE_INTERFACE_NAME,
+            service_class=nims.session_management.GRPC_SERVICE_CLASS,
+        )
+    )
+
+    with reserve_session(
+        session_management_client,
+        measurement_service.context.pin_map_context,
+        timeout=60,
+        pin_names=pin_names,
+    ) as reservation:
+        if len(reservation.session_info) != 1:
+            measurement_service.context.abort(
+                grpc.StatusCode.INVALID_ARGUMENT,
+                f"Unsupported number of sessions: {len(reservation.session_info)}",
+            )
+
+        session_grpc_channel = measurement_service.get_channel(
+            provided_interface=nidcpower.GRPC_SERVICE_INTERFACE_NAME,
+            service_class="ni.measurementlink.v1.grpcdeviceserver",
+        )
```

Comment:
I think we should refactor this too.

You could write a `get_grpc_device_channel(measurement_service, driver_module)`. Usage: `get_grpc_device_channel(measurement_service, nidcpower)`. Inside, it would use `getattr(driver_module, "GRPC_SERVICE_INTERFACE_NAME")`.

Reply:
Refactored this code as suggested